tests/fdb/args/fdb_args.py

```diff
@@ -0,0 +1,21 @@
+# FDB MAC Expire Args file
+
+def add_fdb_mac_expire_args(parser):
+    """
+        Adding arguments required for FDB MAC expire test cases
+
+       Args:
+            parser: pytest parser object
+
+        Returns:
+            None
+    """
+    fdb_group = parser.getgroup("FDB test suite options")
+
+    fdb_group.addoption(
+        "--fdb_aging_time",
+        action="store",
+        type=int,
+        default=60,
```

Comment:
I think the default should be 600 seconds

Reply:
Ansible test does change the default timeout to be 60 seconds [here](https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/test/tasks/fdb_mac_expire.yml#L9) for testing purpose. The default value is restored from the backup copy of switch.json.